pkgs/by-name/au/autologin/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromSourcehut,
+  meson,
+  ninja,
+  pam,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "autologin";
+  version = "1.0.0";
+
+  src = fetchFromSourcehut {
+    owner = "~kennylevinsen";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Cy4v/1NuaiSr5Bl6SQMWk5rga8h1QMBUkHpN6M3bWOc=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    meson
+    ninja
+  ];
+  buildInputs = [ pam ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Run a command inside of a new PAM user session";
+    homepage = "https://sr.ht/~kennylevinsen/autologin";
+    changelog = "https://git.sr.ht/~kennylevinsen/autologin/refs/${version}";
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ beviu ];
+    mainProgram = "autologin";
```

Comment:
```suggestion
  meta = {
    description = "Run a command inside of a new PAM user session";
    homepage = "https://sr.ht/~kennylevinsen/autologin";
    changelog = "https://git.sr.ht/~kennylevinsen/autologin/refs/${version}";
    license = lib.licenses.gpl3Plus;
    platforms = lib.platforms.linux;
    maintainers = with lib.maintainers; [ beviu ];
    mainProgram = "autologin";
```

Reply:
(Especially because you wrote `changelog = "... ${version}"`) and someone may accidentally turn this into `lib.version` when migrating to `finalAttrs` pattern (not sure if that will happen though)